### PR TITLE
Use Location problem solved

### PR DESCRIPTION
**Pull request by NitinBhasneria**
Fixes : #305

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

publicLabEditor

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by NitinBhasneria** [comment]
@VladimirMikulic @sagarpreet-chadha there is a error in grunt jasmine showing
`Fatal error: Callback must be a function` please guide

**Comment by VladimirMikulic** [comment]
@NitinBhasneria I suspect that your tests are failing because you changed the function calls from `editor.mapModule.blurredLocation.geocodeWithBrowser(true);` to `myFunctionLocation()`.

**Comment by NitinBhasneria** [comment]
@VladimirMikulic Do you have any solution for this as it is necessary to change the call function.

**Comment by VladimirMikulic** [comment]
Unfortunately, I don't. Your best bet is to look at the callstack & tests and try to pinpoint the problem.

**Comment by NitinBhasneria** [comment]
Ok thanks

**Comment by NitinBhasneria** [comment]
@cesswairimu @nstjean @jywarren ppease review

**Comment by cesswairimu** [comment]
@NitinBhasneria could you please checkoutbthe Travis failures?  Thanks

**Comment by NitinBhasneria** [comment]
@cesswairimu In grunt jasmine, the error is `Fatal error: Callback must be a function
`
and on Travis this is :
Screenshot from 2020-03-29 21-18-55
Also, I don't think this is the error due to pr as my other PR's are having the same error. What do you think?

**Comment by cesswairimu** [comment]
@NitinBhasneria I see this is happening to all of the recent PRs opened an issue for this here #439

**Comment by NitinBhasneria** [comment]
@VladimirMikulic Can you this PR and approve.

File: examples/index.html [hunk 1]
```diff
@@ -335,7 +335,18 @@ <h4><b>By entering co-ordinates :</b></h4>
     </div>
 
     <script>
-     
+      
+      function myFunctionLocation() {
```
Comment (VladimirMikulic) [inline examples/index.html]: `myFunctionLocation` is not an appropriate name for this function. `locateUserHandler` would be better.

**Comment by NitinBhasneria** [comment]
@jywarren This is ready.

**Comment by cesswairimu** [comment]
@NitinBhasneria @govindgoel @VladimirMikulic apparently as it is now...we are not experiencing this issue...must have been solved on another issue and the issue was never closed....kindly confirm this

**Comment by NitinBhasneria** [comment]
Thank you @cesswairimu I checked it again but in branch master, the issue is still there.

**Comment by jywarren** [comment]
I think this looks good to merge; but maybe what @cesswairimu is saying is that this may not be occurring on PublicLab.org -- this fix is for the demo just in the editor repository. Could that be the ambiguity?

**Comment by jywarren** [comment]
Thanks all, but let's file/open a follow-up if there are ambiguities remaining! Thanks!!! 🎉

**Pull request merged**